src/storage/mvcc/txn.rs

```diff
@@ -2472,7 +2360,12 @@ mod tests {
             );
 
             // If the lock is not expired, cleanup will return the lock info.
-            assert_lock_info_eq(must_cleanup_err(&engine, k, 10, 1), &expected_lock_info);
+            let inner = *cleanup::tests::must_err(&engine, k, 10, 1).0;
+            if let crate::storage::txn::ErrorInner::Mvcc(e) = inner {
```

Comment:
Use match rather than `if let` if there is an `else` branch. Also need to import `storage::txn::ErrorInner`.